lib/material/extra/input_chips/material_input_chips.flow

```diff
@@ -116,15 +137,19 @@ MInputChips(items : Transform<[string]>, style : [MInputChipsStyle]) -> Material
 				TScrollAxisEnabled(const(true), const(false))
 			]);
 	
-	MBaselineCols2(caption, chips)
+	MBaselineCols([caption, chips])
 	|> (\m -> MAttachWidth(m, leftWidth))
 	|> (\m -> if (multiline) MAttachWidth(m, fullWidth) else MBaselineCols2(m, autoComplete))
 	|> (\m -> MAttachAvailableWidth(m, avWidth))
 	|> (\m -> eitherMap(maxWidth, \mw -> MAvailable(m, TFillWYT(mw.width)), m))
 	|> (\m -> MFrame(0., 0., [MFill(MWhite())], m))
-	|> (\m -> MConstruct([makeSubscribe2(items, \itms ->
-			nextDistinct(selectedItems, filter(getValue(selectedItems), \it -> contains(itms, it)))
-		)], m))
+	|> (\m -> MConstruct(
+		[
+			makeSubscribe(selectedItems, \i -> nextDistinct(selectedItems, uniq(i))),
```

Comment:
Better to update with delay. Also, add a style for this behaviour, it's not always an expected one.